nglview/tests/test_utils.py

```diff
@@ -191,3 +192,11 @@ def test_js_utils():
     js_utils.hide_toolbar()
     js_utils.show_toolbar()
     js_utils.execute('print("hello")')
+
+
+def test_compare_images():
```

Comment:
skip the test if not having `cv2` and `skimage`
e.g: https://github.com/arose/nglview/blob/master/nglview/tests/test_widget.py#L279